packages/monitor-v2/test/OptimisticGovernorMonitor.ts

```diff
@@ -448,4 +496,361 @@ describe("OptimisticGovernorMonitor", function () {
     assert.isTrue(spyLogIncludes(spy, 0, toUtf8String(explanation)));
     assert.equal(spy.getCall(0).lastArg.notificationPath, "optimistic-governor");
   });
+  it("Automatically execute transactions", async function () {
+    // Deploy a new OG module proxy and approve the proposal bond.
+    const space = "test.eth";
+    const { customAvatar, ogModuleProxy } = await deployOgModuleProxy({
+      space,
+      quorum: 0,
+      votingPeriod: 0,
+    });
+    await bondToken.connect(proposer).approve(ogModuleProxy.address, await ogModuleProxy.getProposalBond());
+
+    // Set supported bond settings.
+    const supportedBonds: SupportedBonds = {};
+    supportedBonds[bondToken.address] = (await ogModuleProxy.getProposalBond()).toString();
+
+    // Fund the avatar
+    await bondToken.mint(customAvatar.address, parseEther("500"));
+
+    // Construct the transaction data for spending tokens from the avatar.
+    const txnData1 = await bondToken.populateTransaction.transfer(await proposer.getAddress(), parseEther("250"));
+    const txnData2 = await bondToken.populateTransaction.transfer(await random.getAddress(), parseEther("250"));
+
+    if (!txnData1.data || !txnData2.data) throw new Error("Transaction data is undefined");
+
+    const operation = 0; // 0 for call, 1 for delegatecall
+
+    // Create the proposal with multiple transactions.
+    const transactions = [
+      { to: bondToken.address, operation, value: 0, data: txnData1.data },
+      { to: bondToken.address, operation, value: 0, data: txnData2.data },
+    ];
+
+    const explanation = toUtf8Bytes("These transactions were approved by majority vote on Snapshot.");
```

Comment:
Sorry, I'm a little confused, does the explanation not typically include some link to the proposal being executed (like an ipfs hash)? I assumed that was part of the validation, but maybe I'm missing something...

Reply:
Ah, understood.